src/v0/destinations/salesforce/transform.js

```diff
@@ -87,12 +89,19 @@ function responseBuilderSimple(
   }
 
   const response = defaultRequestConfig();
-  const header = {
-    'Content-Type': JSON_MIME_TYPE,
-    Authorization: authorizationData.token,
-  };
+  const finalHeader =
+    authorizationFlow === 'oauth'
+      ? {
+          'Content-Type': JSON_MIME_TYPE,
+          Authorization: `Bearer ${authorizationData.token}`,
+        }
+      : {
+          'Content-Type': JSON_MIME_TYPE,
+          Authorization: authorizationData.token,
+        };
```

Comment:
```suggestion
  const getAuthHeader = (authInfo) => {
    const { authorizationFlow, authorizationData } = authInfo; 
    return authorizationFlow === 'oauth'
      ? { Authorization: `Bearer ${authorizationData.token}` }
      : { Authorization: authorizationData.token };
  }
```
Use this `getAuthHeader` method as follows


```javascript
response.headers = {
   Content-Type: JSON_MIME_TYPE,
   ...getAuthHeader({authorizationFlow, authorizationData}),
}
```